src/go/options/configgenerator.go

```diff
@@ -155,7 +155,7 @@ func DefaultConfigGeneratorOptions() ConfigGeneratorOptions {
 
 	return ConfigGeneratorOptions{
 		CommonOptions:                           DefaultCommonOptions(),
-		BackendDnsLookupFamily:                  "auto",
+		BackendDnsLookupFamily:                  "v4preferred",
```

Comment:
Update the comment for default value in https://github.com/GoogleCloudPlatform/esp-v2/blob/a8d79f086947cce47695d8e49fc25d6f8c5c3bb1/src/go/configmanager/flags/flags.go#L43